apps/io-wallet-user-func/tsup.config.js

```diff
@@ -6,5 +6,5 @@ export default defineConfig({
   sourcemap: true,
   clean: true,
   minify: false,
-  target: "node18",
+  target: "node20",
```

Comment:
I think it should also be updated [.nvmrc ](https://github.com/pagopa/io-wallet/blob/master/.nvmrc)

Reply:
do we want to delete the file? do we use nvm?

Reply:
I think `nodenv` should be used